pages/about/index.js

```diff
@@ -65,7 +59,7 @@ const TEAM_MEMBERS = [
     name: "Chrispin Pullen",
     title: "Senior Application Analyst",
     pronouns: "",
```

Comment:
@patrickm02L i'm just nitpicking for the sake of it. if you wanted to leave this code in better shape, imo it would be better to have `pronouns` be `null` instead of an empty string. that feels more explicit than relying on `{props.pronouns && <p>{pronouns}</p>}` in L208 to handle the empty string as falsey.

Reply:
I'll make a mental note to do this when we add the two new GIS analysts the site. I've asked Diana for their photos and info